test/Constraints/mutating_members_compat.swift

```diff
@@ -6,31 +6,31 @@ struct Foo {
   mutating func boom() {}
 }
 
-let x = Foo.boom // expected-warning{{partial application of 'mutating' method}}
+let x = Foo.boom // expected-warning{{cannot reference 'mutating' method as function value; calling the function has undefined behavior and will be an error in future Swift versions}}
```

Comment:
I am not sure why this test was working on main before? the warnings for swift 4 should have always had the second clause. 

But alas, run the tests locally and fixed the CI issues